modules/wim_features/wim_core/wim_core.strongarm.inc

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @file
+ * wim_core.strongarm.inc
+ */
+
+/**
+ * Implements hook_strongarm().
+ */
+function wim_core_strongarm() {
+  $export = array();
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'admin_theme';
+  $strongarm->value = 'seven';
+  $export['admin_theme'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'theme_default';
```

Comment:
Having a default theme set in a feature is not the best option. Some municipalities will have a custom theme. Feature will be overridden all the time